src/shogun/classifier/AveragedPerceptron.cpp

```diff
@@ -67,9 +67,12 @@ bool CAveragedPerceptron::train_machine(CFeatures* data)
 	while (!(CSignal::cancel_computations()) && (!converged && iter<max_iter))
 	{
 		converged=true;
+		SG_INFO("Iteration Number : %d\n", iter);
```

Comment:
In fact for the perceptron I would not write anything in info, just the iteration number